ldevnet/ldevnet.go

```diff
@@ -28,20 +28,20 @@ import (
 	"github.com/filecoin-project/lotus/node/repo"
 	"github.com/filecoin-project/lotus/storage/sbmock"
 	"github.com/ipfs/go-datastore"
-	crypto "github.com/libp2p/go-libp2p-crypto"
-	peer "github.com/libp2p/go-libp2p-peer"
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
 	mocknet "github.com/libp2p/go-libp2p/p2p/net/mock"
 	"github.com/stretchr/testify/require"
 )
 
 func init() {
 	build.SectorSizes = []uint64{1024}
 	build.MinimumMinerPower = 1024
+	os.Setenv("TRUST_PARAMS", "1")
 }
 
 type LocalDevnet struct {
```

Comment:
From here below, some extra hacks to code inherited from Lotus with some changes to make it useful for us.